Enterprise-CMCS/macpro-mdct-mcr
# MLR Redline Updates

### Description

@oweingrod-coforma and I went through the latest MLR Figma files with Kim and found a couple of necessary changes, which are addressed in this PR. Most of the updates are related to verbiage and styling.

### Related ticket(s)

MDCT-2392

---
### How to test

Option 1:
Go through the MLR report and check the app matches Figma (don't worry about actually going through everything mockup by mockup, we did that fun part)

Option 2:
Make sure things look nice, links open in new tabs, and there are no typos 💯 

### Important updates

The `mlr.json` file went through significant changes.

### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
